syntax = "proto3";
message TestMsg {
	string msg = 1;
	int32 index = 2;
	repeated string person = 3;
}